core/io/resource_format_binary.h

```diff
@@ -165,6 +168,7 @@ class ResourceFormatSaverBinaryInstance {
 		FORMAT_FLAG_NAMED_SCENE_IDS = 1,
 		FORMAT_FLAG_UIDS = 2,
 		FORMAT_FLAG_REAL_T_IS_DOUBLE = 4,
+		FORMAT_FLAG_HAS_SCRIPT_CLASS = 4,
```

Comment:
Duplicate value. I assume it's not intended.

Reply:
oh, my mistake